Modules/Journey/tabs/QuestsByZone/QuestsByZoneTab.lua

```
@@ -128,7 +142,64 @@ _CreateZoneDropdown = function()
+    local professionIds = zoneMap[QuestieJourney.questCategoryKeys.PROFESSIONS]
+    if professionIds then
+        for profId in pairs(professionIds) do
+            local profQuests = zoneMap[profId]
+            if profQuests then
+                for questId in pairs(profQuests) do
+                    allQuestIds[questId] = true
+                end
+            end
+        end
+    -- add all quest IDs from pet battle quests
+    local petQuests = zoneMap[QuestieDB.sortKeys.PET_BATTLE]
```

Luacheck (1.2.0)

[error] expected 'end' (to close 'if' on line 165) near 'local' (indentation-based guess)

(E011)